src/web/routes/static/static-router.ts

```diff
@@ -2,24 +2,8 @@ import { Router, static as Static } from 'express';
 
 import { join } from 'path';
 
-import {
-	jiraAdminOnlyAuthorizationMiddleware,
-	jiraContextSymmetricJwtFromQueryAuthenticationMiddleware,
-} from '../../middleware/jira';
-
 export const staticRouter = Router();
 
-staticRouter.use(
-	'/admin/index.html',
```

Comment:
**note:** The auth issue is not reproducible with the admin page since this page is served from the `/static/admin` but not `static/admin/index.html` URL (see [here](https://github.com/atlassian-labs/figma-for-jira/blob/5489ea7b0d6088f75833a51c15cfa6e03b80dc99/src/atlassian-connect.ts#L109)) and therefore, the auth middlewares have no impact.

Reply:
🤦🏻‍♂️ the confirmation bias in me testing this code